src/javaharness/java/arcs/android/demo/service/ArcsService.java

```diff
@@ -97,8 +97,10 @@ public void startArc(
       }
 
       @Override
-      public void stopArc(String arcId) {
-        // TODO(csilvestrini): Implement
+      public void stopArc(String arcId, String pecId) {
+        // TODO: Stop the running arc once the Arcs Runtime supports that.
+
+        pecPortManager.removePecPort(pecId);
```

Comment:
probably we could at least also call `arc.dispose()` here? what else should that do?